### PR TITLE
Base the RHEL9 CIS preview on RHEL8

### PR DESCRIPTION
**Pull request by matejak**
The previous "RHEL9 CIS profile" was the projection of the former RHEL8 profile, so it was quite misleading.

This PR harnesses our policy files to get a RHEL9 projection of the current RHEL8 CIS.

Another option could be to remove the profile files completely, as unlike e.g. ANSSI profiles, CIS is released for a particular product, not for a general Linux system that is then interpreted on a product-by-product basis. And a rela CIS benchmark for RHEL9 doesn't exist.

### CODE REVIEW

**Comment by matusmarhefka** [comment]
@matejak We will need also kickstart files.

**Review by yuumasato — APPROVED** [review]
LGTM.

**Comment by yuumasato** [comment]
@matejak There are rules missing CCEs. From the ctest logs:
```
xccdf_org.ssgproject.content_rule_accounts_password_pam_pwhistory_remember_password_auth
xccdf_org.ssgproject.content_rule_accounts_password_pam_pwhistory_remember_system_auth
xccdf_org.ssgproject.content_rule_root_path_no_dot
xccdf_org.ssgproject.content_rule_file_permissions_efi_grub2_cfg
xccdf_org.ssgproject.content_rule_sshd_set_loglevel_verbose
xccdf_org.ssgproject.content_rule_sshd_set_maxstartups
```

File: products/rhel9/kickstart/ssg-rhel9-cis_workstation_l1-ks.cfg [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+# Create primary system partitions (required for installs)
+part /boot --fstype=xfs --size=512
+part pv.01 --grow --size=1
+
+# Create a Logical Volume Management (LVM) group (optional)
+volgroup VolGroup --pesize=4096 pv.01
+
+# Create particular logical volumes (optional)
+logvol / --fstype=xfs --name=LogVol06 --vgname=VolGroup --size=10240 --grow
```
Comment (matusmarhefka) [inline products/rhel9/kickstart/ssg-rhel9-cis_workstation_l1-ks.cfg:109]: As we don't have other partitions, what about increasing this to make it bigger? Minimal disk size requirement should be 20 GiB so I think we can go with the initial `size=16896`. The same would apply also for the server l1.

File: products/rhel8/kickstart/ssg-rhel8-cis-ks.cfg [hunk 2]
```diff
@@ -106,7 +106,7 @@ part pv.01 --grow --size=1
 volgroup VolGroup --pesize=4096 pv.01
 
 # Create particular logical volumes (optional)
-logvol / --fstype=xfs --name=LogVol06 --vgname=VolGroup --size=10240 --grow
+logvol / --fstype=xfs --name=LogVol06 --vgname=VolGroup --size=16896 --grow
```
Comment (matusmarhefka) [inline products/rhel8/kickstart/ssg-rhel8-cis-ks.cfg:109]: This should remain 10240.

File: products/rhel8/kickstart/ssg-rhel8-cis_workstation_l2-ks.cfg [hunk 3]
```diff
@@ -106,7 +106,7 @@ part pv.01 --grow --size=1
 volgroup VolGroup --pesize=4096 pv.01
 
 # Create particular logical volumes (optional)
-logvol / --fstype=xfs --name=LogVol06 --vgname=VolGroup --size=10240 --grow
+logvol / --fstype=xfs --name=LogVol06 --vgname=VolGroup --size=16896 --grow
```
Comment (matusmarhefka) [inline products/rhel8/kickstart/ssg-rhel8-cis_workstation_l2-ks.cfg:109]: This should also remain 10240.

File: products/rhel9/kickstart/ssg-rhel9-cis-ks.cfg [hunk 4]
```diff
@@ -106,7 +106,7 @@ part pv.01 --grow --size=1
 volgroup VolGroup --pesize=4096 pv.01
 
 # Create particular logical volumes (optional)
-logvol / --fstype=xfs --name=LogVol06 --vgname=VolGroup --size=10240 --grow
+logvol / --fstype=xfs --name=LogVol06 --vgname=VolGroup --size=16896 --grow
```
Comment (matusmarhefka) [inline products/rhel9/kickstart/ssg-rhel9-cis-ks.cfg:109]: This should remain 10240.

File: products/rhel9/kickstart/ssg-rhel9-cis_workstation_l2-ks.cfg [hunk 5]
```diff
@@ -106,7 +106,7 @@ part pv.01 --grow --size=1
 volgroup VolGroup --pesize=4096 pv.01
 
 # Create particular logical volumes (optional)
-logvol / --fstype=xfs --name=LogVol06 --vgname=VolGroup --size=10240 --grow
+logvol / --fstype=xfs --name=LogVol06 --vgname=VolGroup --size=16896 --grow
```
Comment (matusmarhefka) [inline products/rhel9/kickstart/ssg-rhel9-cis_workstation_l2-ks.cfg:109]: And the same, it should remain 10240.

**Review by matusmarhefka — CHANGES_REQUESTED** [review]

**Review by matusmarhefka — APPROVED** [review]

**Pull request merged**